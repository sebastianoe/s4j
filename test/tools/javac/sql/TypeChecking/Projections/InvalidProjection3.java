/*
 * @test
 * @compile/fail/ref=InvalidProjection3.out InvalidProjection3.java -sqlrepo /Users/sebastian/ma/jdk7u-langtools/test/tools/javac/sql/repo.json
 */
// key: compiler.err.sql.validation.types.not.compatible
import java.sql.ResultSet;

public class InvalidProjection3 {
	public static void main(String[] args) {
		Integer i = SQL[SELECT first_name FROM customer LIMIT 1];
	}
}