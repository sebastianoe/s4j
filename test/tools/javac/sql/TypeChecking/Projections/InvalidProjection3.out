error: The determined sql query type java.lang.String cannot be assigned to the expected type java.lang.Integer
1 error